fvm/src/syscalls/gas.rs

```diff
@@ -21,6 +22,6 @@ pub fn charge_gas(
         .map(|_| ())
 }
 
-pub fn available(context: Context<'_, impl GasOps>) -> Result<u64> {
+pub fn available(context: Context<'_, impl Kernel>) -> Result<u64> {
```

Comment:
This syscall will _hopefully_ go away and turn into a read of a global variable. See https://github.com/filecoin-project/FIPs/discussions/845